### PR TITLE
Bem sass

### PR DESCRIPTION
**Pull request by gorzat**
image

### CODE REVIEW

**Review by matzimowski — COMMENTED** [review]
screenshot or never happened!

**Comment by matzimowski** [comment]
🐥

File: 04 - CSS JS parser/colorStyles.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <h2>Subtitle <span class='blue pink'>is</span> here</2>
+  </header>
+  
+  <main>
+    <p>Some text <i class='pink yellow'>which</i> is sometimes <strong class="magenta blue">bold!</strong></p>
+  </main>
+  
+  <footer>
+    <table>
+      <tr><td class='yellow green'>a</td><td class='magenta red'>b</td></tr>
```
Comment (vergilius) [inline 04 - CSS JS parser/colorStyles.html:22]: `"` vs `'` - decide :)

File: 04 - CSS JS parser/colorStyles.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>JS Bin</title>
+  <script src="colorStyles.js"></script>
```
Comment (matzimowski) [inline 04 - CSS JS parser/colorStyles.html:7]: maybe `script` could be included just before closing `</html>`?

File: 04 - CSS JS parser/colorStyles.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+</head>
+<body>
+  
+  <header class="green">
```
Comment (vergilius) [inline 04 - CSS JS parser/colorStyles.html:11]: in terms of semantics, your css classes should be more descriptive

File: 04 - CSS JS parser/colorStyles.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+const root = document.body;
+
+
+//root.children.length root.children[i]
+//root.parentElement
+//elem.classList.add/remove/contains
+//elem.style.backgroundColor 
+//$0.style[name] = value;
+
+function addStyle(styleCss, el, colorEl){
```
Comment (vergilius) [inline 04 - CSS JS parser/colorStyles.js:11]: please fix whitespaces over here

Comment (vergilius) [reply 04 - CSS JS parser/colorStyles.js:11]: and below

**Review by matzimowski — APPROVED** [review]
bem bem bem 💥

**Review by vergilius — APPROVED** [review]
JS could get some more <3

**Pull request merged**